### ISSUE TITLE
Navigating through the event pages should persist location to URL

### ISSUE THREAD

**Issue by caesay** [exceptionless/Exceptionless.UI#133]
This sits on the back of #132, once that is complete we can look at this:

The current view of events should be entirely represented in the URL (search string, page number, anchor element etc). The reason why this is important / useful is being able to send a link to a colleague and have them be looking at the same page as you instead of needing to walk them through the search string, links to click, page number to be on, etc.

**Comment by niemyjski** [comment]
Yeah, I really want this ability too. It's really tough to implement this right now because everything is a component and some dashboards have two lists on it. What do you know what page goes to which component? I guess our component could have a paging prefix (might be a simple fix). That might make this easy to implement.

**Comment by caesay** [comment]
I don't know about any challenges that are angular specific, but identifying what url param goes to what component shouldn't be a big pain. As far as I know we only have a "Frequent" feed and a "Recent" feed, right? so we just need to prefix between the two like you say.

**Comment by niemyjski** [comment]
"Most views in the UI allow 20 items per page, and you get to cycle between pages. A frequent use case for me is to look at all my sessions, and then drill into a session to see the events. If I move beyond page 1 of my session events, drill into a session, and then press the browser back button to return to the list... I'm returned to page 1, not the page I was just on. This makes drilling into details very cumbersome for any events not listed on the first page." @boydpatterson
